docs/fleet/_index.md

```diff
@@ -18,103 +18,62 @@ aliases:
+- obtain health metrics, such as status, uptime, version, or [logs](machines/#logs)
+- perform debugging
+
+All of this is possible when you are close to your machine, as well as remotely from anywhere in the world.
 
-To facilitate collaboration, you can add collaborators to organizations, manage access, and share locations across multiple organizations.
+## Use Viam for collaboration
 
 When you create a Viam account, Viam automatically creates an organization for you.
 You can use this organization as your collaboration hub by inviting collaborators to your organization.
 You can also add additional organizations as desired at any time.
 
-{{< alert title="Caution" color="caution" >}}
-Everyone you invite as an owner to your organization has complete access to everything in that organization.
-This includes the permissions to delete machines and locations, as well as the ability to remove you from the organization.
-{{< /alert >}}
-
-You can also share locations across different organizations **that you are part of**.
-
-### Permissions
-
-You can assign users different fleet management capabilities by making them an owner or an operator of a given {{< glossary_tooltip term_id="organization" text="organization" >}}, {{< glossary_tooltip term_id="location" text="location" >}}, or {{< glossary_tooltip term_id="machine" text="machine" >}}.
+To facilitate collaboration, you can grant individual collaborators or entire organizations granular permissions for individual machines or entire locations.
+This allows you flexibility to manage internal machines, sell devices to external customers and keep managing them, and collaborate with different partners or companies on groups of machines.
 For more information, see [Permissions](/fleet/rbac/#permissions).
 
-## Collaborate on your machines
-
-Viam is built in a way that allows you to change configurations, deploy packages, check logs, and control your machines both when you are close to your machine, as well as remotely.
-
-Machine [configuration](machines/#configuration) and machine [code](#control-with-the-sdks) is intentionally kept separate, allowing you to keep track of versioning and debug issues separately.
-
 ### Configuration
 
-Everyone who has access to the location the machine is in, can change the machine's [configuration](machines/#configuration).
+When you or your collaborators change the configuration of a machine or a group of machines in the Viam app, `viam-server` automatically synchronizes the configuration and updates the running resources within 15 seconds.
+This means has access, can change a fleet's [configuration](machines/#configuration), even while your machines are running.
+
+You can see configuration changes made by yourself or by your collaborators on the [History tab](machines/#history).
+You can also revert to an earlier configuration from the History tab.
 
 {{< alert title="Simultaneous config edits" color="caution" >}}
```

Comment:
I feel this should be removed from the overview page. But should exist elsewhere in the docs

Reply:
It does exist on the configure page. I do feel it's kind of important to call out when we talk about collaboration because it's an unexpected gotcha.